pages/api/sum.ts

```diff
@@ -0,0 +1,9 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+import { sum } from "@/pages/utils/sum";
+
+export default function handler(req: NextApiRequest, res: NextApiResponse) {
+  const { num1, num2 } = req.query;
+  const result = sum(num1, num2);
```

Comment:
```
const sum = calculateSum(num1, num2)
res.status(200).json({ sum })
```

Reply:
Can you provide more context to this? or what do you really mean to do with this?